### PR TITLE
Mahavir backend

### PR DESCRIPTION
**Pull request by MahavirSkeps**
Added index file in src and also updated middleware folder multer.js file

### CODE REVIEW

File: backend/src/middleware/multer.js [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+const path=require('path')
 
-module.exports = upload;
\ No newline at end of file
+const storage = multer.diskStorage({
+    destination: (req, file, cb) => {
+        console.log("req",req);
+        console.log("reqfile",file);
+        cb(null, '/Users/utkarsh.nigam/Desktop/NodeJS Practice/POC-data-Extractor/service_data_extractor/backend/src/uploads');
+        console.log("hello");
```
Comment (ntnmangla1) [inline backend/src/middleware/multer.js]: remove console statement

File: backend/src/middleware/multer.js [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+const multer=require('multer')
+const path=require('path')
 
-module.exports = upload;
\ No newline at end of file
+const storage = multer.diskStorage({
+    destination: (req, file, cb) => {
+        console.log("req",req);
+        console.log("reqfile",file);
+        cb(null, '/Users/utkarsh.nigam/Desktop/NodeJS Practice/POC-data-Extractor/service_data_extractor/backend/src/uploads');
```
Comment (ntnmangla1) [inline backend/src/middleware/multer.js]: is this require local path?

**Review by ntnmangla1 — CHANGES_REQUESTED** [review]

File: backend/src/middleware/multer.js (hunk 2 above)
Comment (MahavirSkeps) [reply backend/src/middleware/multer.js]: yes

File: backend/src/middleware/multer.js (hunk 1 above)
Comment (MahavirSkeps) [reply backend/src/middleware/multer.js]: done

Comment (ntnmangla1) [reply backend/src/middleware/multer.js]: remove console.log(hello) also

**Review by ntnmangla1 — CHANGES_REQUESTED** [review]

**Review by ntnmangla1 — APPROVED** [review]
lftm

File: backend/src/middleware/multer.js [hunk 3]
```diff
@@ -1,3 +1,16 @@
-const upload = ("");
+const multer=require('multer')
+const path=require('path')
 
-module.exports = upload;
\ No newline at end of file
+const storage = multer.diskStorage({
+    destination: (req, file, cb) => {
+        cb(null, '/Users/utkarsh.nigam/Desktop/NodeJS Practice/POC-data-Extractor/service_data_extractor/backend/src/uploads');
```
Comment (pradeep-sRawat) [inline backend/src/middleware/multer.js]: use relative path not absolute path

**Review by pradeep-sRawat — CHANGES_REQUESTED** [review]

File: backend/src/middleware/multer.js (hunk 3 above)
Comment (MahavirSkeps) [reply backend/src/middleware/multer.js]: changed

**Review by pradeep-sRawat — APPROVED** [review]

**Pull request merged**